### PR TITLE
Feature/instrument fields

### PR DESCRIPTION
**Pull request by rodrigopavezi**
@donnapep @stulees @tejohnso @settinghead please review. cheers

### CODE REVIEW

**Comment by settinghead** [comment]
:+1:

File: rise-financial.js [hunk 1]
```diff
@@ -22,13 +22,23 @@ var financialVersion = "1.0.0";
...
             type: String,
             value: ""
+          },
+
+          /**
+           * The list of instruments fields the component should return data for
+           */
+          instrumentFields: {
+            type: Array,
+            value: function value() {
```
Comment (stulees) [inline rise-financial.js]: @rodrigopavezi You can use ES6 `=>` here instead

**Review by stulees — CHANGES_REQUESTED** [review]

File: rise-financial.js (hunk 1 above)
Comment (stulees) [inline rise-financial.js]: @rodrigopavezi Actually, disregard

**Review by stulees — APPROVED** [review]

File: rise-financial.js (hunk 1 above)
Comment (rodrigopavezi) [inline rise-financial.js]: @stulees thanks!!!

File: rise-financial-es6.js [hunk 2]
```diff
@@ -10,13 +10,23 @@
...
           type: String,
           value: ""
+        },
+
+        /**
+         * The list of instruments fields the component should return data for
+         */
+        instrumentFields: {
+          type: Array,
+          value: () => {
```
Comment (stulees) [inline rise-financial-es6.js]: @rodrigopavezi Ya my bad, disregard

**Pull request merged**